src/kernels/periodic.jl

```diff
@@ -0,0 +1,15 @@
+"""
+PeriodicKernel()
```

Comment:
According to the [Julia documentation](https://docs.julialang.org/en/latest/manual/documentation/), one should write this as
```suggestion
    PeriodicKernel

```

> so that it is printed as Julia code.

Alternatively, of course, one could specify the function signature of the constructor, but I guess that would involve either `dims` or `r`?

Reply:
Corrected!